Session/Conversations/ConversationVC.swift

```diff
@@ -1241,6 +1271,28 @@ final class ConversationVC: BaseVC, ConversationSearchControllerDelegate, UITabl
     }
 
     // MARK: - General
+    
+    func addOrRemoveOutdatedClientBanner(contactIsUsingOutdatedClient: Bool) {
+        // Do not show the banner until the new disappearing messages is enabled
+        guard Features.useNewDisappearingMessagesConfig else { return }
+        
+        guard contactIsUsingOutdatedClient else {
+            UIView.animate(
+                withDuration: 0.25,
+                animations: { [weak self] in
+                    self?.outdatedClientBanner.alpha = 0
+                },
+                completion: { [weak self] _ in
+                    self?.outdatedClientBanner.alpha = 1
+                    self?.outdatedClientBanner.removeFromSuperview()
+                }
+            )
+            return
+        }
+
+        self.view.addSubview(self.outdatedClientBanner)
+        self.outdatedClientBanner.pin([ UIView.HorizontalEdge.left, UIView.VerticalEdge.top, UIView.HorizontalEdge.right ], to: self.view)
```

Comment:
Looks like this covers the top of the tableView which results in the empty state (or top-most message) being obscured by the banner

Reply:
Looks like this one is still an issue:

<img width="410" alt="Screenshot 2023-08-04 at 11 02 24 am" src="https://github.com/oxen-io/session-ios/assets/15862619/38c1b579-42ff-41d1-80df-e8b84c4817c2">

Reply:
<img width="453" alt="image" src="https://github.com/oxen-io/session-ios/assets/8543570/9e8ba16b-d346-4d59-bbe6-5fec5c5a591a">
Fixed!